tket-qsystem/tests/guppy_opt.rs

```diff
@@ -44,19 +45,39 @@ fn guppy_simple_cx() -> Hugr {
     load_guppy_circuit("simple_cx")
 }
 
+fn count_gates(h: &impl HugrView) -> (usize, usize) {
```

Comment:
Could be nice to have a comment explaining what this is counting and why.

As we dicussed in person it could be nice to have a map with keys as OpType and number of instances of the optypes values